CONTRIBUTING.md

```diff
@@ -21,6 +21,23 @@ maximize the chances of your PR being merged.
 
 * See [STYLE.md](STYLE.md)
 
+# Inclusive language policy
+
+The Envoy community has an explicit goal to be inclusive to all. As such, all PRs must adhere to the
+following guidelines for all code, APIs, and documentation:
+
+* The following words and phrases are not allowed:
+  * *Whitelist*: use allowlist instead.
```

Comment:
`alllowlist` or `allow-list` or `allow list`. Is the former used by any other projects?

Reply:
From my unofficial research I see `allowlist` being used, but I don't feel very strongly about it.

Reply:
One example from envoy: https://github.com/envoyproxy/envoy/blob/f87e68f8a7eef471b1e12b8ef1f662ab845356fb/api/envoy/extensions/filters/http/grpc_stats/v3/config.proto#L32

Reply:
Yep, looking around it seems common enough, SGTM.